config/missing-page-redirector.php

```diff
@@ -85,6 +84,8 @@
         '/userList.php' => '/users',
         '/history.php' => '/user/{u}/history',
         '/historyexamine.php' => '/user/{u}/history/{d}',
+        '/usergameactivity.php' => '/user/{f}/game/{ID}',
+        '/gamecompare.php' => '/user/{f}/game/{ID}/compare',
```

Comment:
This change, or the changes in `recent-game-players.blade.php` are necessary before deployment. It was missed when applying feedback in #2245.

I can create a separate PR if it seems unlikely that this will be merged.